### ISSUE TITLE
Actualiser le site pour SOTM FR 2022

### ISSUE THREAD

**Issue by naomap** [osm-fr/sotmfr-website#39]
-  Créer le sous-domaine sotm2022.openstreetmap.fr
- Changer le visuel
- Restaurer le menu
- Intégrer le formulaire Framaforms pour l'appel à contribution

**Comment by naomap** [comment]
Des volontaires parmi @BaptisteDixneuf @Alketta @ThomasG77 @tomao44 ?

**Comment by vinber** [comment]
en amont, @jocelynj créer l'espace sur un bout de serveur et diriger le nouveau nom de domaine ?

**Comment by jocelynj** [comment]
Yep, je peux ajouter sotm2022.openstreetmap.fr sur la VM qui héberge actuellement les sotm précédents. Ça vous va de partir de la même VM ?

**Comment by vinber** [comment]
> Yep, je peux ajouter sotm2022.openstreetmap.fr sur la VM qui héberge actuellement les sotm précédents. Ça vous va de partir de la même VM ?

Je crois que cela ne pose pas de souci, il n'y a, à ma connaissance, pas de révolution de prévue

**Comment by jocelynj** [comment]
Le site "est en place": http://sotm2022.openstreetmap.fr/

Reste à configurer côté VM.

**Comment by vinber** [comment]
merci @ThomasG77 

j'ai collé le lien pour le formulaire dans la PR.
J'ai les droits pour faire Merge mais comme je ne comprends rien je ne suis pas sur que cela soit le plus pertinent :D

Qui a les droits, comprends ou qui veut les droits pour le faire ?

**Comment by ThomasG77** [comment]
@vinber Mis à jour OK sur https://labs.webgeodatavore.com/sotmfr-2022/.
Si tu veux, on peut faire ensemble par écran interposé pour le "merge", genre Jitsi, dans le week-end.

- [ ] Revoir la formulation de chaque page
- [ ] J'ai raboté un peu trop le menu en cachant le contenu. Peut être faire réapparaître les menus cachées...

Attente de vos retours

**Comment by vinber** [comment]
yep, cela va être chaud pour moi, famille toussa. Peut-être diffuser sur sotm ton lien pour avis et puis après on envoie !

et on améliorera.

1 point 

- [ ] Nous recherchons des volontaires ! Faire le lien sur le https://forum.openstreetmap.fr/t/simpliquer-dans-lorganisation-du-sotm-fr/8135

**Comment by vinber** [comment]
et franchement après je pense que l'on peut publier en l'état 

@naomap @vdct je trouve pas d'autres à pinguer

**Comment by naomap** [comment]
C'est super merci @ThomasG77.

Pour moi c'est bon à publier, ça permettra de remettre un cuop de com' pour l'appel à contributions.

**Comment by ThomasG77** [comment]
>  Nous recherchons des volontaires ! Faire le lien sur le https://forum.openstreetmap.fr/t/simpliquer-dans-lorganisation-du-sotm-fr/8135

Le lien pour l'appel à volontaires est maintenant sur l'accueil (cf capture ci- dessous)

![Sélection_645](https://user-images.githubusercontent.com/642120/158030413-b8c50704-b6d1-43b5-b155-754e83563584.png)

> J'ai les droits pour faire Merge mais comme je ne comprends rien je ne suis pas sur que cela soit le plus pertinent :D

@vinber Donne-moi les droits de mainteneur sur le dépôt https://github.com/osm-fr/sotmfr-website, stp pour que je "merge".

On verra ultérieurement pour publier automatiquement vers http://sotm2022.openstreetmap.fr depuis le dépôt. A priori, à voir avec @jocelynj

**Comment by vinber** [comment]
@jocelynj je ne trouve plus comment donner les droits à @ThomasG77 pour qu'il puisse merger. J'suis un boulet

**Comment by ThomasG77** [comment]
@vinber https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository ?

**Comment by gendy54** [comment]
Oui on peut publier. Ca fera plus propre que pas de site du tout

**Comment by tonyemery** [comment]
Du côté des moyens d'accès, il faudrait ajouter :
- l'arrivée/départ depuis l'aéroport ;
- le covoiturage

**Comment by jocelynj** [comment]
@vinber: J'ai ajouté les permissions pour @ThomasG77 sur ce repository.

**Comment by vinber** [comment]
> @vinber https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository ?

en fait, je suis "collaborator" mais alors que sur un autre dépot (celui des attributions) je peux gérer les membres et tutti, ici je ne peux pas ...

@jocelynj peux-tu me donner un peu plus de possibilités stp ?

**Comment by vinber** [comment]
merci c'est ok

**Comment by jocelynj** [comment]
@vinber: je t'ai mis Admin sur ce repository. Tu devrais pouvoir maintenant gérer les permissions des utilisateurs.
